api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md

```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
 - *"series"* - data fields go to the chart series.
 
 ---
+The **customizeChart**, **customizeSeries**, and **processCell** fields of the **integrationOptions** object are functions. They accept arguments whose structure is described below.
+
+- **customizeChart**
+
+        <!--JavaScript-->customizeChart: function (chartOptions) {
+            // Change chart properties here
+            return chartOptions; // This line is optional
+        }
+        
+    The **chartOptions** object has the following structure:
+
+    <table class="dx-table full-width">
+        <tr>
+            <th>Field name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>valueAxis</b></td>
+            <td>Array</td>
+            <td>Allows you to customize the automatically-created value axes. Each object in this array can contain fields described in the <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/" target="_blank">valueAxis</a> property description. When this array holds several objects, use their <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name" target="_blank">name</a> field to identify the axis.</td>
+        </tr>
+        <tr>
+            <td><b>panes</b></td>
+            <td>Array</td>
+            <td>Allows you to customize the automatically-created panes. Each object in this array can contain fields described in the <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/" target="_blank">panes</a> property description. When this array holds several objects, use their <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/#name" target="_blank">name</a> field to identify the pane.</td>
+        </tr>
+    </table>
+
+    You can also specify any other [Chart properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/) in this object, except [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource).
+
+- **customizeSeries** 
+
+        <!--JavaScript-->customizeSeries: function (seriesName, seriesOptions) {
+            // Change series properties here
+            return seriesOptions; // This line is optional
+        }
+
+    The **seriesName** argument is a string that allows you to identify a series. The **seriesOptions** argument is an object with the following structure:
+
```

Comment:
The passed object already has values in the following members and allows you to customize the automatically generated Chart settings